plugin.json

```diff
@@ -129,11 +129,17 @@
           "help_text": "Set the buffer size for streaming files from MS Teams to Mattermost",
           "default": 20
         },{
-            "key": "connectedUsersAllowed",
-            "display_name": "Max Connected Users",
-            "type": "number",
-            "help_text": "The maximum number of users that may connect their MS Teams account. Once connected, the user is added to a white-list and may reconnect at any time.",
-            "default": 1000
+          "key": "connectedUsersAllowed",
+          "display_name": "Max Connected Users",
+          "type": "number",
+          "help_text": "The maximum number of users that may connect their MS Teams account. Once connected, the user is added to a white-list and may reconnect at any time.",
+          "default": 1000
+        },{
+          "key": "connectedUsersInvitePoolSize",
+          "display_name": "Connection Invites: Max Pending Invitations",
+          "type": "number",
+          "help_text": "Invite pool size: the maximum number of connection invites that may be pending at a given time. When enabled, connection invite DMs will be sent to users as they login, up to the maximum specified here. As invited users connect, spaces in the invite pool will open up and more invites will be sent out. (Leave empty to disable connection invites.)",
```

Comment:
By "enabled" or "empty", do we mean "non-zero" or "0"? 0/5 on the language, just wanting to clarify.

Reply:
Updated to "When specified, ..." and "Set to 0 or leave empty to..."